Framework/Algorithms/inc/MantidAlgorithms/FilterEvents.h

```diff
@@ -74,6 +74,8 @@ class DLLExport FilterEvents : public API::Algorithm {
   }
 
 private:
+  friend struct Mantid::Kernel::CreateUsingNew<FilterEvents>;
```

Comment:
This is not useful at all. It was from a failed attempt.  But I forgot to delete.